launch/start.launch.xml

```diff
@@ -66,9 +66,7 @@
     <include file="$(find-pkg-share realsense2_camera)/launch/rs_launch.py">
       <arg name="align_depth" value="true"/>
       <arg name="enable_pointcloud" value="true"/>
-      <arg name="color_width" value="640"/>
-      <arg name="color_height" value="360"/>
-      <arg name="color_fps" value="90.0"/>
+      <arg name="rgb_camera.profile" value="640x360x90"/>
```

Comment:
Does this actually work? @DavidSpielman tried this with a real camera and it complained about the resolution dimensions. I think it wanted "640x480x30"

Reply:
It seems to achieve 640x360 on the D455 cameras mounted on the demos. I guess it isn't valid on the D435i? I'll lower the FPS down to 30.